README.md

```diff
@@ -77,6 +77,23 @@ to match how the VA operates in production and staging environments. If you use
 a host-based firewall (e.g. `ufw` or `iptables`) make sure you allow connections
 from the Docker instance to your host on the required ports.
 
+If you run Boulder's clients in docker containers and want to connect
+to the Boulder instance started by docker-compose, you may attach your
+containers to the boulder_default network that docker-compose creates
+and tell Boulder to use the DNS resolver provided by Docker and not the
+fake one. For that change `dnsResolver` in `test/config/va.json` and
+`test/config/va.json` to point to `127.0.0.11:53`. Then after
+`docker-compose up` you can use `http://boulder:4000/directory` in your
+clients to connect to Boulder as long as they are attached to
+boulder_default network. Alternatively you can change
+docker-compose.yml to attach the Boulder container to the docker network
+where your containers run.
+
+If you use this Boulder instance to connect only from Docker containers
```

Comment:
I think this would read clearer as:
"If you only connect to the Boulder instance from other Docker containers and not from ..."